.github/actions/npm-release/action.yml

```
@@ -25,9 +25,16 @@ runs:
       run: |
         git config --global user.email "${{ inputs.git_email }}"
         git config --global user.name "${{ inputs.git_username }}"
-    - name: 'Bump version and commit'
+    - name: 'Bump version'
       shell: bash
-      run: npm version ${{ inputs.releaseType }} -m '[skip ci] Publish release %s'
+      run: npm version ${{ inputs.releaseType }} --no-git-tag-version
```

I think we need `run` here?

```suggestion
      run: npm run version ${{ inputs.releaseType }} --no-git-tag-version
```

---

There is actually a command called [`npm version`](https://docs.npmjs.com/cli/v9/commands/npm-version). It knows how to bump versions and whatnot and commit those changes (unless you disabled that with a flag as we have here).

As part of that command run, it executes certain [lifecycle scripts](https://docs.npmjs.com/cli/v10/using-npm/scripts#npm-version). So if you have defined a `version` command in the `scripts` section of your `package.json`, it will automatically be invoked. There's also a `preversion` hook. But you don't need to call `npm run version` yourself. I agree it's confusing that the command name and lifecycle script hooks have the same name.

---

Ah, good to know. Thank you!